### DISCUSSION TITLE
Multiple sessions

### DISCUSSION THREAD

**Discussion by DaruinHerrera** [open-wa/wa-automate-nodejs#2548] [General]
How can I have multiple active sessions, that is, have two WhatsApp numbers connected online answering

**Comment by smashah** [comment]
You can just use `create` multiple times if you're doing this inside one process

**Reply by DaruinHerrera** [reply]
Can you tell me as I'm new to nodejs

**Reply by smashah** [reply]
@DaruinHerrera If you show some code then I can show you how to do it
